src/sectors.jl

```diff
@@ -267,21 +305,21 @@ Return the Frobenius-Schur indicator of a sector `a`.
 """
 function frobeniusschur(a::Sector)
     return if FusionStyle(a) isa UniqueFusion || FusionStyle(a) isa SimpleFusion
-        sign(Fsymbol(a, conj(a), a, a, leftone(a), rightone(a)))
+        sign(Fsymbol(a, dual(a), a, a, leftunit(a), rightunit(a)))
     else
-        sign(Fsymbol(a, conj(a), a, a, leftone(a), rightone(a))[1])
+        sign(Fsymbol(a, dual(a), a, a, leftunit(a), rightunit(a))[1])
     end
 end
 
 # Not necessary
 function Asymbol(a::I, b::I, c::I) where {I <: Sector}
     return if FusionStyle(I) isa UniqueFusion || FusionStyle(I) isa SimpleFusion
         (sqrtdim(a) * sqrtdim(b) * invsqrtdim(c)) *
-            conj(frobeniusschur(a) * Fsymbol(dual(a), a, b, b, leftone(a), c))
+            dual(frobeniusschur(a) * Fsymbol(dual(a), a, b, b, leftunit(a), c))
```

Comment:
```suggestion
            conj(frobeniusschur(a) * Fsymbol(dual(a), a, b, b, leftunit(a), c))
```
This one is acting on a `Number` 😉